packages/pluggableWidgets/accordion-web/CHANGELOG.md

```diff
@@ -3,14 +3,17 @@ All notable changes to this widget will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [unreleased]
+## [1.1.0] - 2021-07-22
```

Comment:
You need to keep the unreleased on top as pattern.

Reply:
Added it